subsys/bluetooth/controller/ll_sw/ull_llcp_local.c

```diff
@@ -110,7 +110,7 @@ static ALWAYS_INLINE uint32_t shared_data_access_lock(void)
 static ALWAYS_INLINE void shared_data_access_unlock(bool key)
 {
 	if (key) {
-		/* We are in thread context and have to reenable TICKER_USER_ID_ULL_HIGH */
+		/* We are in thread context and have to re-enable TICKER_USER_ID_ULL_HIGH */
```

Comment:
Curious that you've change re-usable to reusable and here you modify reenable to re-enable, when in both cases, both spellings are valid